committeeoversightapp/models.py

```diff
@@ -51,6 +51,19 @@ def latest_rating(self):
         rating_set = self.committeerating_set.order_by('-congress')
         return rating_set[0]
 
+
+    def sliced_name(self):
+        chambers = {
```

Comment:
This is very clever, but I'm finding it a little hard to follow! Any reason a simpler approach, e.g., [`string.replace`](https://docs.python.org/3.6/library/stdtypes.html#str.replace) or [`re.sub`](https://docs.python.org/3.6/library/re.html#re.sub), wouldn't work?

Reply:
Does switching out `split` for `replace` address your concern, or did you want to see a deeper change to the logic?

Reply:
I mean the dictionary and iteration are a bit circuitous. You could achieve this more directly like

```python
if self.parent.name in ('United States House of Representatives', 'United States Senate'):
    return self.name.replace('House Committee on ', '').replace('Senate Committee on ', '')
```

or even

```python
# if parent.name...
return re.sub(r'(House|Senate) Committee on ', '', self.name)
```